cms/jinja2/templates/components/navigation/breadcrumbs.html

```diff
@@ -12,6 +12,9 @@
             {% endif %}
         {% endif %}
     {% endfor %}
+    {% if page.include_self_in_breadcrumbs %}
```

Comment:
This doesn't look used anywhere? It would be good to either clean up, or make use of it and add some dev comments for context